client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -169,19 +162,4 @@ trait ContentApiQueries {
   val restaurantReviews = RestaurantReviewsQuery()
   val filmReviews = FilmReviewsQuery()
   val videoStats = VideoStatsQuery()
-  def next[Q <: PaginatedApiQuery[Q]](q: Q, id: String) = NextQuery(normalize(q), id)
-
-  private def normalize[Q <: PaginatedApiQuery[Q]]: Q => Q =
-    normalizePageSize andThen normalizeOrder
-
-  private def normalizePageSize[Q <: PaginatedApiQuery[Q]]: Q => Q = 
-    q => if (q.has("page-size")) q else q.pageSize(10)
-
-  private def normalizeOrder[Q <: PaginatedApiQuery[Q]]: Q => Q = 
-    q => if (q.has("order-by")) 
-      q 
-    else if (q.has("q"))
-      q.orderBy("relevance")
-    else
-      q.orderBy("newest")
```

Comment:
The `normalize` code is now implemented by `PaginatedApiQuery.setPaginationConsistentWith(response)`